src/Oro/Bundle/EntityExtendBundle/Migration/Extension/ExtendExtension.php

```diff
@@ -904,6 +904,19 @@ public function getFieldNameByColumnName($tableName, $columnName)
         return $this->entityMetadataHelper->getFieldNameByColumnName($tableName, $columnName);
     }
 
+    /**
+     * Gets a field name by a table name and a column name
+     *
+     * @param string $className
+     * @param string $fieldName
+     *
+     * @return string|null
+     */
+    public function getColumnNameByFieldName($className, $fieldName)
```

Comment:
Can you show an example when this method is useful?
for me, it can make some mess in migrations as currently we always work there with table names and column names, but with this method, there will be mix with class names and field names. It looks less readable.

Reply:
When adding enums, you can't get the field name :

```
$this->extendExtension->addEnumField(...)
```

In fact, it is using the 3rd argument + `_id`

There is no method for removing an enum nor any method to build this relation field name, only line in the code. Should we assume it is a convention ?